car/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN gcc -o gps_uploader -g -fsanitize=undefined -fno-sanitize-recover src/gps_up
 # Some demo example GPS data
 COPY testsuite /testsuite
 COPY gps_data.txt /testsuite/gps_data.txt
-COPY gps_data.txt .
+
+ENV AFL_INST_LIBS=0
```

Comment:
May not be necessary, added as this can help with shorter examples where we want to find defects quickly, so we don't get bogged down in libc code.